docs/source/developing-volttron/quick-start.rst

```diff
@@ -0,0 +1,331 @@
+       remote: Counting objects: 100% (4892/4892), done.
+       remote: Compressing objects: 100% (1971/1971), done.
+       remote: Total 82987 (delta 3155), reused 4294 (delta 2890), pack-reused 78095
+       Receiving objects: 100% (82987/82987), 102.73 MiB | 4.19 MiB/s, done.
+       Resolving deltas: 100% (57997/57997), done.
+       Checking out files: 100% (1807/1807), done.
+       ...
+
+.. note::
+
+   In this tutorial we download the VOLTTRON code to the default home directory. 
+   However, feel free to download the code to a different place as desired.
+
+.. code-block:: bash
+
+       # $ mkdir <path-to-dir>
+       # $ cd <path-to-dir>
+       # $ git clone https://github.com/VOLTTRON/volttron
+
+After successfully downloading the VOLTTRON package, change the current working path to the code path. Then, inspect the source code files.
+
+.. code-block:: bash
+
+       $ cd volttron
+       $ ls
+
+.. code-block:: bash
+
+       # expected output similar to this
+       bootstrap.py     deprecated    pylintrc          requirements.py  stop-volttron
+       ci-integration   docs          pytest.ini        scripts          TERMS.md
+       CONTRIBUTING.md  examples      README.md         services         volttron
+       COPYRIGHT        integrations  readthedocs.yml   setup.py         volttron_data
+       debugging_utils  LICENSE.md    RELEASE_NOTES.md  start-volttron   volttrontesting
+
+3. Bootstrap VOLTTRON environment
+------------------------------
+
+VOLTTRON is a Python-based platform. In this step, we will rely on the :code:`bootstrap.py` script in the root directory to bootstrap the platform environment. This process will create a virtual environment and install the package's Python dependencies.
+
+.. note::
+
+   VOLTTRON provides different message bus options. In this tutorial we will demonstrate the default ZeroMQ option. (Read more about :ref:`message bus<Message-Bus>`.)
+
+
+Bootstrap the VOLTTRON environment by running the following command. (This may take a while.)
+
+.. code-block:: bash
+
+       $ python3 bootstrap.py
+
+.. code-block:: bash
+
+       # expected output similar to this
+       UPDATE: []
+       Installing required packages
+       + pip install --no-deps wheel==0.30
+       Collecting wheel==0.30
+       Using cached
+       <https://files.pythonhosted.org/packages/0c/80/16a85b47702a1f47a63c104c91abdd0a6704ee8ae3b4ce4afc49bc39f9d9/wheel-0.30.0-py2.py3-none-any.whl>
+       ...
+
+
+After bootstrap finished, we activate the Python virtual environment with the following command:
+
+.. code-block:: bash
+
+       $ source env/bin/activate
+
+You may notice the command prompt has changed and there is the virtual environment name as prefix. e.g., :code:`(volttron) user@host:~/volttron $`. The prefix environment name indicates the virtual environment is activated.
+
+Alternatively, you can use the following command to verify if the virtual environment is up.
```

Comment:
This assumes that they don't have a different virtual environment setup such as if you have a conda environment then you already have a virtual env setup.

Reply:
Resolve: Added the following note-block
![image](https://user-images.githubusercontent.com/28743873/158310966-79d8c24e-df38-48d3-8812-eef5e1081ec2.png)